torch/_inductor/codecache.py

```diff
@@ -825,7 +825,15 @@ def __init__(
                     from triton.runtime.autotuner import Autotuner
 
                     kernel = kernel_side_table.get_kernel(node.kwargs["kernel_idx"])
+                    configs = None
                     if isinstance(kernel, Autotuner):
+                        if kernel.configs:
+                            configs = str(
+                                sorted(
```

Comment:
are these values always sortable?

Reply:
Probably most of the times, but let me `str` them to be on the safe side.